test/integration/query.test.ts

```diff
@@ -86,6 +86,10 @@ describe('integration tests', () => {
 
     expect(teams).toHaveLength(1);
     expect(teams[0].name).toBe('Team fruits');
+
+    const serialized = teams.toSerializable();
```

Comment:
Should we add a test for `toString` after this to make sure both serialisation and deserialisation work together?

Reply:
Sure